CHANGELOG.md

```diff
@@ -5,6 +5,18 @@
 - [Amazon Linux 2 release notes](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-al2.html)
 - [Amazon Linux AMI 2018.03 Release Notes](https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/) 
 
+## 20230906
+- ecs version: 1.75.3-1
+- source al1 ami: amzn-ami-minimal-hvm-2018.03.0.20230807.0-x86_64-ebs
```

Comment:
Some versions of source AMIs listed in the changlog do not match our `release.auto.pkrvars.hcl` 
```
source_ami_al1                = "amzn-ami-minimal-hvm-2018.03.0.20230821.0-x86_64-ebs"
source_ami_al2                = "amzn2-ami-minimal-hvm-2.0.20230822.0-x86_64-ebs"
source_ami_al2arm             = "amzn2-ami-minimal-hvm-2.0.20230822.0-arm64-ebs"
source_ami_al2kernel5dot10    = "amzn2-ami-minimal-hvm-2.0.20230822.0-x86_64-ebs"
source_ami_al2kernel5dot10arm = "amzn2-ami-minimal-hvm-2.0.20230822.0-arm64-ebs"
source_ami_al2023             = "al2023-ami-minimal-2023.1.20230825.0-kernel-6.1-x86_64"
source_ami_al2023arm          = "al2023-ami-minimal-2023.1.20230825.0-kernel-6.1-arm64"
```

Reply:
thx for catching that, fixing now